contrib/connector/fc/linuxfc.go

```diff
@@ -176,3 +176,28 @@ func trimDoubleQuotesInText(str string) string {
 	}
 	return str
 }
+
+func getMultipathDevice(deviceWWN string) (string, error) {
```

Comment:
Also, do we check somewhere if multipath is enabled on the host or not? Will it be a better idea to check if /etc/multipath conf exists and then if yes, check if the device has multipath

Reply:
this code doesn't care if multipath enabled or not on host, it tries to get multipath device first, if not exist, then tries to find normal sd device instead.
But I alse think it would better add some kind of configuration item to allow user to specify whether or not using mulitipath.
I'll think about it.

Reply:
True, we can give some config for enable/disabling this feature, we can open an issue for this and think about how we can do, thanks

Reply:
Ok. So this code can be additional check. We can go with it. For multipath enabled/disabled, i think Host already provides the /etc/multipath file if multipath is enabled or not. we can use that.